token/bearer.go

```diff
@@ -15,30 +15,43 @@ import (
 )
 
 var (
-	errNilBearerToken     = errors.New("bearer token is not set")
 	errNilBearerTokenBody = errors.New("bearer token body is not set")
 	errNilBearerTokenEACL = errors.New("bearer token ContainerEACL table is not set")
 )
 
+// BearerToken represents bearer token for object service operations.
 type BearerToken struct {
 	token acl.BearerToken
 }
 
-// ToV2 converts BearerToken to v2 BearerToken message.
+// ReadFromV2 reads BearerToken from the acl.BearerToken message.
 //
-// Nil BearerToken converts to nil.
-func (b *BearerToken) ToV2() *acl.BearerToken {
-	if b == nil {
-		return nil
-	}
+// See also WriteToV2.
+func (b *BearerToken) ReadFromV2(m acl.BearerToken) {
+	b.token = m
+}
 
-	return &b.token
+// WriteToV2 writes Decimal to the accounting.Decimal message.
+// The message must not be nil.
+//
+// See also ReadFromV2.
+func (b BearerToken) WriteToV2(m *acl.BearerToken) {
+	*m = b.token
 }
 
-func (b *BearerToken) Empty() bool {
-	return b == nil || b.token.GetBody() == nil && b.token.GetSignature() == nil
+// Empty returns true if bearer token has no fields set.
+func (b BearerToken) Empty() bool {
+	return b.token.GetBody() == nil && b.token.GetSignature() == nil
 }
 
+// SetLifetime sets bearer token lifetime parameters. They are the same as
+// JWT lifetime parameters but measured in NeoFS epochs.
+//
+// 'exp' stands for token expiration epoch.
+//
+// 'nbf' stands for 'valid not before' epoch.
+//
+// 'iat' stands for 'issued at` epoch.
 func (b *BearerToken) SetLifetime(exp, nbf, iat uint64) {
```

Comment:
```suggestion
//
// See also Expiration, NotBeforeTime and IssuedAt.
func (b *BearerToken) SetLifetime(exp, nbf, iat uint64) {
```

Reply:
Not relevant after https://github.com/nspcc-dev/neofs-sdk-go/pull/185#discussion_r848308124